src/launcher/index.ts

```diff
@@ -227,7 +228,10 @@ async function main() {
     try {
       flowrWindow = createFlowrWindow(store, isDebugMode, setDebugMode)
       keyboard.setParentWindow(flowrWindow)
-      FullScreenManager.applyDefaultActionOnWindow(flowrWindow)
+      if (!defaultBrowserWindowOptions(store).kiosk) {
```

Comment:
Do not use setResizable/setFullscreenable is Kiosk mode. This blocks resizing and end up in black margins